libs/ngx-sherlock/src/lib/derivable-input.decorator.test.ts

```diff
@@ -0,0 +1,103 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { atom, SettableDerivable } from '@skunkteam/sherlock';
+import { DerivableInput } from './derivable-input.decorator';
+import { NgxSherlockModule } from './ngx-sherlock.module';
+
+describe(DerivableInput, () => {
+    @Component({
+        selector: 'sherlock-test-component',
+        template: `{{ mapped$ | value }}`,
+        changeDetection: ChangeDetectionStrategy.OnPush,
+    })
+    class TestComponent {
+        @Input() @DerivableInput() input$!: SettableDerivable<string>;
+
+        readonly mapped$ = this.input$.map(s => `Hello ${s}!`).fallbackTo('Got nothing!');
```

Comment:
Or, because we can...
```suggestion
        readonly mapped$ = template`Hello ${this.input$}!`.fallbackTo('Got nothing!');
```

with: `import { template } from '@skunkteam/sherlock-utils';`